resources/benchmark/tpcds/schema.sql

```diff
@@ -0,0 +1,552 @@
+create table dbgen_version
```

Comment:
The filename should clarify that this is only used for sqlite. Same for the foreign keys.

Reply:
This table has actually no relevant purpose for us.